indico/modules/events/timetable/client/js/actions.ts

```diff
@@ -146,8 +147,33 @@ export function setDraftEntry(data): SetDraftEntryAction {
   return {type: SET_DRAFT_ENTRY, data};
 }
 
-export function moveEntry(date: string, entries: TopLevelEntry[]): MoveEntryAction {
-  return {type: MOVE_ENTRY, date, entries};
+export function moveEntry(entry, eventId, entries: TopLevelEntry[], date: string) {
+  let entryURL: string;
+
+  switch (entry.type) {
+    case 'break':
+      entryURL = breakURL({event_id: eventId, break_id: entry.objId});
+      break;
+    case 'block':
+      entryURL = sessionBlockURL({event_id: eventId, session_block_id: entry.objId});
+      break;
+    default:
+      entryURL = contributionURL({event_id: eventId, contrib_id: entry.objId});
+  }
+
+  let entryData;
+
+  if (entry.type === 'contrib') {
+    entryData = {start_dt: moment(entry.startDt).format('YYYY-MM-DDTHH:mm:ss')};
+  } else {
+    entryData = {start_dt: entry.startDt};
+  }
+
+  return ajaxAction(() => indicoAxios.patch(entryURL, snakifyKeys(entryData)), null, () => ({
```

Comment:
The data is already in snake_case, so I'd either change to camelCase or remove the call.